### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pgebheim**
Add explainer of Bonus Tokens

### CODE REVIEW

File: transactions/burn-flow/2022/sep-21/README.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # Burn Flow - Sep 21
 
 The Flow Service account will burn 38,357,187 FLOW after it has been
-transferred from the Genesis Token Account. These tokens are accounted for as
-part of the FlowToken.totalSupplybut are *not currently considered* as part of
+transferred from the Genesis Token Account. These tokens are "Bonus Tokens" and
```
Comment (laynelafrance) [inline transactions/burn-flow/2022/sep-21/README.md:4]: can we remove this transfer note? It's confusing/needlessly concerning

File: transactions/burn-flow/2022/sep-21/README.md [hunk 2]
```diff
@@ -1,10 +1,17 @@
 # Burn Flow - Sep 21
 
 The Flow Service account will burn 38,357,187 FLOW after it has been
-transferred from the Genesis Token Account. These tokens are accounted for as
-part of the FlowToken.totalSupplybut are *not currently considered* as part of
+transferred from the Genesis Token Account. These tokens are "Bonus Tokens" and 
+are accounted for as part of the FlowToken.totalSupply but are *not currently considered* as part of
```
Comment (laynelafrance) [inline transactions/burn-flow/2022/sep-21/README.md:5]: update to "not considered"
```suggestion
are accounted for as part of the FlowToken.totalSupply but are *not considered* as part of
```

"currently" implies that will change which it won't. These tokens won't exist after they are burnt and there won't again be a notion of bonus tokens.

File: transactions/burn-flow/2022/sep-21/README.md (hunk 1 above)
Comment (laynelafrance) [inline transactions/burn-flow/2022/sep-21/README.md:4]:
```suggestion
. These tokens are "Bonus Tokens" and 
```

File: transactions/burn-flow/2022/sep-21/README.md [hunk 3]
```diff
@@ -1,10 +1,17 @@
 # Burn Flow - Sep 21
 
 The Flow Service account will burn 38,357,187 FLOW after it has been
```
Comment (laynelafrance) [inline transactions/burn-flow/2022/sep-21/README.md:3]:
```suggestion
The Flow Service account will burn 38,357,187 FLOW
```

File: transactions/burn-flow/2022/sep-21/README.md [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
-transferred from the Genesis Token Account. These tokens are accounted for as
-part of the FlowToken.totalSupplybut are *not currently considered* as part of
+transferred from the Genesis Token Account. These tokens are "Bonus Tokens" and 
+are accounted for as part of the FlowToken.totalSupply but are *not currently considered* as part of
 the FlowToken inflation while paying rewards.
 
+## About Bonus Tokens
+
+During the initial Flow launch two batches of tokens were initially created by the Gensis Account. These consisted the initial Flow token supply and a second batch of 65mm tokens which are referred to as "Bonus Tokens." These 65mm tokens were withheld from the Flow Total Supply and distributed as locked tokens, only to be unlocked if the Flow mainnet launch milestones were reached. These milestomnes were all achieved, and the 65mm Bonus Tokens are all reclaimed and will be destroyed. These locked tokens were not in circulation as they were not transferrable by the holding parties, and were not considered for totalSupply inflation as part of the rewards calculation.
```
Comment (laynelafrance) [inline transactions/burn-flow/2022/sep-21/README.md:10]:
```suggestion
During the initial Flow launch two batches of tokens were created by the Gensis Account. These consisted of the initial Flow token supply and a second batch of 65mm tokens which are referred to as "Bonus Tokens." These 65mm tokens were withheld from the Flow Total Supply and distributed as locked tokens, only to be unlocked if the Flow mainnet launch milestones were NOT reached. These milestones were all achieved, and the 65mm Bonus Tokens can now be destroyed. These locked tokens were not in circulation as they were not transferrable by the holding parties, and were not considered for totalSupply inflation as part of the rewards calculation.
```

**Review by laynelafrance — CHANGES_REQUESTED** [review]

File: transactions/burn-flow/2022/sep-21/README.md (hunk 1 above)
Comment (bluesign) [reply transactions/burn-flow/2022/sep-21/README.md]: Agreed someone can ask why not burning 65m? Did they earned rewards? Where is the rest? 

I suggest keep transparency at minimum level ( foundation is good example for that ) 

Flow service account will burn 38M to promote Flow ecosystem is great!

**Pull request merged**